src/fides/api/ops/models/privacy_experience.py

```diff
@@ -32,88 +35,214 @@ class DeliveryMechanism(Enum):
+            history_data.pop("updated_at")
+            history_data["experience_config_id"] = self.id
+
+            PrivacyExperienceConfigHistory.create(
+                db, data=history_data, check_name=False
+            )
+
+        return self
+
+    def dry_update(self, *, data: dict[str, Any]) -> PrivacyExperienceConfig:
+        """
+        A utility method to get an updated object without saving it to the db.
+
+        This is used to see what an object update would look like, in memory,
+        without actually persisting the update to the db
+        """
+        cloned_attributes = self.__dict__.copy()
+        for key, val in data.items():
+            cloned_attributes[key] = val
+        cloned_attributes.pop("_sa_instance_state")
+        return PrivacyExperienceConfig(**cloned_attributes)
+
+    @hybridproperty
+    def experience_config_history_id(self) -> Optional[str]:
+        """Convenience property that returns the experience config id for the current version.
+
+        Note that there are possibly many historical records for the given experience config, this just returns the current
+        corresponding historical record.
+        """
+        history: PrivacyExperienceConfigHistory = self.histories.filter_by(  # type: ignore # pylint: disable=no-member
+            version=self.version
+        ).first()
+        return history.id if history else None
+
+
+class PrivacyExperienceConfigHistory(ExperienceConfigBase, Base):
+    """Experience Config History - stores the history of how the config changed"""
+
+    experience_config_id = Column(
+        String, ForeignKey(PrivacyExperienceConfig.id_field_path), nullable=False
+    )
+
+
+class PrivacyExperienceBase:
+    """Base Privacy Experience fields that are common between privacy experiences and historical records"""
 
+    disabled = Column(Boolean, nullable=False, default=False)
+    component = Column(EnumColumn(ComponentType), nullable=False)
+    delivery_mechanism = Column(EnumColumn(DeliveryMechanism), nullable=False)
+    region = Column(EnumColumn(PrivacyNoticeRegion), nullable=False, index=True)
     version = Column(Float, nullable=False, default=1.0)
-    privacy_experience_template_id = Column(
-        String, ForeignKey(PrivacyExperienceTemplate.id_field_path), nullable=True
+
+
+class PrivacyExperience(PrivacyExperienceBase, Base):
+    """Stores Privacy Experiences for a given just a single region.
+    There can only be one component per region.
+    """
+
+    experience_config_id = Column(
+        String,
+        ForeignKey(PrivacyExperienceConfig.id_field_path),
+        nullable=True,
+        index=True,
+    )
+
+    experience_config_history_id = Column(
+        String,
+        ForeignKey(PrivacyExperienceConfigHistory.id_field_path),
+        nullable=True,
+        index=True,
+    )  # Also links to the historical record, so if the version of config gets updated, that
+    # triggers a new version of the experience.
```

Comment:
I thought it made sense to link both the ExperienceConfig and the ExperienceConfigHistoryId to the PrivacyExperience.    

- experience_config_id makes it easier to fetch `PrivacyExperienceConfig.experiences` - all experiences that currently use this PrivacyExperienceConfig 
- `experience_config_history_id` makes it easier to bump versions.  If the ExperienceConfig itself is updated, its version is bumped, which I think should trigger that the PrivacyExperience's version is likewise incremented.  The PrivacyExperience is now showing different language or features potentially.